### PR TITLE
Remove HQT from OHF

### PR DESCRIPTION
**Pull request by Yantom1**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/text-generation/utils.py [hunk 1]
```diff
@@ -187,41 +187,30 @@ def get_torch_compiled_model(model):
...
+    except ImportError:
+        raise ImportError(
+            "Module neural_compressor is missing. Please use a newer Synapse version to use quantization."
+        )
 
-        habana_quantization_toolkit.prep_model(model)
+    config = FP8Config.from_json_file(args.quant_config)
+    if config.measure:
+        model = prepare(model, config)
+    if config.quantize or args.assistant_model is not None:
```
Comment (ssarkar2) [inline examples/text-generation/utils.py:200]: "or args.assistant_model is not None" is extra from previous. Just making sure that is intended?

File: examples/image-to-text/README.md [hunk 2]
```diff
@@ -74,7 +74,15 @@ python3 run_pipeline.py \
...
-Inference for Llava-1.5-7b, Llava-1.5-13b, Llava-v1.6-mistral-7b and Llava-v1.6-vicuna-13b in FP8 precision are enabled using the Quantization Toolkit (HQT), which provides model measurement and quantization capabilities in PyTorch.
+Inference for Llava-1.5-7b, Llava-1.5-13b, Llava-v1.6-mistral-7b and Llava-v1.6-vicuna-13b in FP8 precision are enabled using the Intel Neural Compressor (INC), which provides model measurement and quantization capabilities in PyTorch.
+Models that have been validated:
+  - [nlpconnect/vit-gpt2-image-captioning](https://huggingface.co/nlpconnect/vit-gpt2-image-captioning)
+  - [Salesforce/blip-image-captioning-large](https://huggingface.co/Salesforce/blip-image-captioning-large)
+  - [Salesforce/blip-image-captioning-base](https://huggingface.co/Salesforce/blip-image-captioning-base)
+
+### Running with FP8
+
+Llava-1.5-7b and Llava-1.5-13b in FP8 are enabled using the Intel Neural Compressor (INC), which provides model measurement and quantization capabilities in PyTorch.
```
Comment (MrGeva) [inline examples/image-to-text/README.md:86]: remove the extra code, this is not part of the change.

**Review by ssarkar2 — APPROVED** [review]

**Comment by regisss** [comment]
> I've made small modification to also remove HQT in image-to-text example @ssarkar2 @regisss Can we please merge this PR?

I'm not sure it works with Synapse1.17, does it?

**Comment by astachowiczhabana** [comment]
Hi @regisss 
This is a synapse 1.18 dependency

**Comment by regisss** [comment]
> Hi @regisss This is a synapse 1.18 dependency

So let's wait for Synapse 1.18 to be released to merge this one.
